src/background.ts

```diff
@@ -4,16 +4,20 @@ import { URLStorage } from "./storage/store_url_array";
 //Creates a Context Menu right click option for saving pdf links
 chrome.contextMenus.removeAll(function() {
     chrome.contextMenus.create({
+        //identifier for right contextmenu item
         id: "rightclickoption",
```

Comment:
camel case this?

Reply:
I think its supposed to be hyphenated I can do that